source/uwp/AdaptiveCardTestApp/Expected/SourceCards/Tests.ActionSet.8bc0461.json

```diff
@@ -0,0 +1,118 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.0",
+	"body": [
+		{
+			"type": "TextBlock",
+			"wrap": true,
+			"text": "Cards can have action sets in the middle of their body."
+		},
```

Comment:
Where us this card coming from?

Reply:
Looks like it's a leftover from when i had action set in the tests directory. I'll remove it.

---
In reply to: [249153452](https://github.com/Microsoft/AdaptiveCards/pull/2244#discussion_r249153452) [](ancestors = 249153452)